rust/operator-binary/src/crd/mod.rs

```diff
@@ -419,6 +456,39 @@ impl v1alpha1::AirflowCluster {
     }
 }
 
+fn extract_role_from_webserver_config(
+    fragment: Role<WebserverConfigFragment>,
+) -> Role<AirflowConfigFragment> {
```

Comment:
Another note for the future, but.. maybe we should look into at least lifting the outline of this logic into op-rs. Again, should look at how hdfs-op does this.

Reply:
Ditto comment re. Druid: when we look at Airflow and Druid together it should be clearer what to lift into the framework.

Reply:
Sure.

Reply:
To be considered in https://github.com/stackabletech/airflow-operator/issues/622